JASP-Desktop/qquick/jasptheme.cpp

```diff
@@ -135,6 +135,9 @@ void JaspTheme::setCurrentTheme(JaspTheme * theme)
 	if(theme == _currentTheme)
 		return;
 
+	if (_currentTheme && theme)
+		theme->setFont(_currentTheme->font());
```

Comment:
Shouldn't the same happen for `_jaspCodeFont`?

Reply:
As in, the init does:
```
	setFont(						_jaspFont);
	setFontLabel(					_jaspFont);
	setFontRibbon(					_jaspFont);
	setFontGroupTitle(				_jaspFont);
	setFontPrefOptionsGroupTitle(	_jaspFont);
	setFontCode(					_jaspCodeFont);
	setFontRCode(					_jaspCodeFont);
```

So why not all of those here?

Reply:
Maybe it is better to make all the fonts static?
There not really supposed to be set from the theme now that we have font-settings right?

Reply:
Yep, and my fix does not work well anyway...